mne/decoding/tests/test_search_light.py

```diff
@@ -29,13 +29,16 @@ def test_search_light():
     from sklearn.pipeline import make_pipeline
     from sklearn.metrics import roc_auc_score, make_scorer
     from sklearn.ensemble import BaggingClassifier
+    from sklearn.base import is_classifier
 
     X, y = make_data()
     n_epochs, _, n_time = X.shape
     # init
     assert_raises(ValueError, SlidingEstimator, 'foo')
     sl = SlidingEstimator(Ridge())
+    assert_true(not is_classifier(sl))
     sl = SlidingEstimator(LogisticRegression())
+    assert_true(is_classifier(sl))
```

Comment:
Should you also check that the resulting `cv` defaults to `StratifiedKFold` and `KFold` in the two cases, respectively?

Reply:
That's a bit hard, cross_val_multiscore is a function not an object, so we can't investigate the way it builds the cv.